pkg/controller/virtualmachinevolumeexport/virtualmachinevolumeexport_controller.go

```diff
@@ -130,7 +130,10 @@ func (r *ReconcileVirtualMachineVolumeExport) getDestination() string {
 	var destination string
 	if r.vmvExport.Spec.Destination.Local != nil {
 		destination = ExporterDestinationLocal
+	} else if r.vmvExport.Spec.Destination.S3 != nil {
+		destination = ExporterDestinationS3
 	}
```

Comment:
need to check if the destination is empty or both local and s3 are filled.

Reply:
thank you for spotting! fixed now